contributors/contribute-to-a-gerrit-project.md

```diff
@@ -61,43 +63,100 @@ You are redirected to Gerrit. Wait a few seconds for the CLA status to update.
+
+1. Read the Confirmation of Association statement and select the checkbox.
 
    ![Gerrit Confirmation of Association](../.gitbook/assets/cla-gerrit-confirmation-of-association.png)
 
-4. Click **CONTINUE**.
+2. Click **CONTINUE**.
 
-   A dialog appears and informs you: You are done!
+   A dialog appears and informs yoy: You are done!
+   
+   ![You are done](../.gitbook/assets/cla-github-you-are-done%20%281%29.png)
 
-5. Click **RETURN TO REPO**.
+3. Click **RETURN TO REPO**.
 
    You are redirected to Gerrit. Wait a few seconds for the CLA status to update or refresh the page.
+   
+## If a **Company has not sigend CCLA** window appears
+
+This window apperas if your comapny has not signed a Corporate CLA with Linux Foundation.
+
+To send an email notification to your company's CLA manager to sign Corporate CLA:
+
+1. Select your email address, that you want your comapny manager to whitelist while signing Corporate CLA, from the **Emial to Authorize** drop-down field.
+
+   ![](/.gitbook/assets/company-not-signed-ccla.png)
+
+2. Click **SEND**.
+
+  A message shows that your email is successfully sent.
+  
+  ![](/.gitbook/assets/email-to-whitelist.png)
+
+## If You are not whitelisted
+
+This window apperas if your comapny has not whitelisted you under their signed Corporate CLA.
+
+To send a request to your company's CLA manager to be whitelisted:
+
+1. Click **CONTACT**.
+
+   ![](/.gitbook/assets/request-to-be-whitelisted.png)
+
+    A **Request Access** window appears.
+  
+2. Select your email address, that you want your comapny manager to whitelist while signing Corporate CLA, from the **Emial to Authorize** drop-down field, and type a message.
 
-   * If you are not associated with a company, click your company name and skip to Step 7 below.
-   * If your company is not associated with a company in the list, continue with the next step.
+3. Click **SEND**
 
-6. Click **COMPANY NOT IN LIST? CLICK HERE**.
+   A message shows that your email is successfully sent.
+  
+  ![](/.gitbook/assets/email-to-whitelist.png)
 
-   A Corporate CLA form appears. Some fields are pre-populated such as the company name and email. Fill out the remaining form fields to contact your CCLA manager to request association to the company. The CCLA manager adds you to the whitelist.
+## If Company is not in the list
 
-   A Confirmation of Association with statement appears. Read the Confirmation of Association statement and mark the agreement checkbox.
+If you don't find your company's name in the list:
 
-7. Click **CONTINUE**.
+1. Click **COMPANY NOT IN LIST? CLICK HERE**.
 
-   A dialog appears and informs you: You are done!
+   The **Verify Your Permission of Access** dialog appears.
+  
+2. Click an answer: Are You a CLA Manager?
 
-8. Click **RETURN TO REPO**.
+   **YES**— You will be reditected to [corporate.lfcla.com](https://corporate.lfcla.com/#/companies) to link your company to signed CLA
```

Comment:
"You will be redirected to [corporate.lfcla.com](https://corporate.lfcla.com/#/companies) to add your company to a project" Link to https://lf-docs-linux-foundation.gitbook.io/easycla/ccla-managers-and-ccla-signatories/add-a-company-to-a-project